homeassistant/components/point/sensor.py

```diff
@@ -45,17 +46,19 @@
 class MinutPointSensor(MinutPointEntity):
     """The platform class required by Home Assistant."""
 
-    def __init__(self, point_client, device_id, device_class):
+    def __init__(self, hass, point_client, device_id, device_class):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.